translations.yml

```diff
@@ -1090,33 +1097,41 @@
 
 - en: 'Editors:'   # gender-neutral, plural. Precedes list of more than one person who edited the resource. If gender-neutral not in your language, use appropriate alternative, such as plural or 'Edition'.
   fr: 'Rédaction :'
+  ja: '編集者：'
 
 - en: 'Update Editor:'   # gender-neutral, singular. Precedes list of one person who updates the resource. If gender-neutral not in your language, use appropriate alternative, such as plural or 'Update Edition'.
   fr: 'Rédaction des mises à jour:'
+  ja: '更新作業者：'
```

Comment:
@hiroya-uga When used, one space is added after this UI term (see footer in https://www.w3.org/WAI/tutorials/images/decision-tree/ for example). The space at the end is therefore redundant.

Reply:
@remibetin It seems no space 👀
Do you mean, I should use half-width (Hankaku) colon instead of full-width (zenkaku)?

> en: Update Editor: Brian Elton
> ja: 更新作業: Brian Elton (half-width + a space + names)
> ja: 更新作業： Brian Elton (full-width + a space + names)

The colon after the Japanese word looks more natural in full-width characters. 
What do you think?

Reply:
@hiroya-uga My bad, I did not notice `：` was a unique character. Everything is fine then!